frontend/components/notifications/SubscriptionDialog.vue

```diff
@@ -0,0 +1,264 @@
+<script setup lang="ts">
```

Comment:
could you please move the two subscription components in the management folder so that all managing stuff is together? 
(sorry I did not see this before)

Reply:
OK 🫡